push_notifications/apns_async.py

```diff
@@ -155,105 +126,103 @@ def _create_notification_request_from_args(
 		topic=None,
 		err_func: ErrFunc = None,
-	) -> APNs:
-		use_sandbox = get_manager().get_apns_use_sandbox(application_id)
-		if topic is None:
-			topic = get_manager().get_apns_topic(application_id)
-		if creds is None:
-			creds = self._get_credentials(application_id)
-
-		client = APNs(
-			**asdict(creds),
-			topic=topic,  # Bundle ID
-			use_sandbox=use_sandbox,
-			err_func=err_func,
-		)
-		return client
-
-	def _get_credentials(self, application_id):
-		if not get_manager().has_auth_token_creds(application_id):
-			# TLS certificate authentication
-			cert = get_manager().get_apns_certificate(application_id)
-			return CertificateCredentials(
-				client_cert=cert,
-			)
-		else:
-			# Token authentication
-			keyPath, keyId, teamId = get_manager().get_apns_auth_creds(application_id)
-			# No use getting a lifetime because this credential is
-			# ephemeral, but if you're looking at this to see how to
-			# create a credential, you could also pass the lifetime and
-			# algorithm. Neither of those settings are exposed in the
-			# settings API at the moment.
-			return TokenCredentials(key=keyPath, key_id=keyId, team_id=teamId)
+) -> APNs:
+	use_sandbox = get_manager().get_apns_use_sandbox(application_id)
+	if topic is None:
+		topic = get_manager().get_apns_topic(application_id)
+	if creds is None:
+		creds = _get_credentials(application_id)
+
+	client = APNs(
+		**asdict(creds),
+		topic=topic,  # Bundle ID
+		use_sandbox=use_sandbox,
+		err_func=err_func,
+	)
+	return client
 
 
-# Public interface
+def _get_credentials(application_id):
+	if not get_manager().has_auth_token_creds(application_id):
+		# TLS certificate authentication
+		cert = get_manager().get_apns_certificate(application_id)
+		return CertificateCredentials(
+			client_cert=cert,
+		)
+	else:
+		# Token authentication
+		keyPath, keyId, teamId = get_manager().get_apns_auth_creds(application_id)
+		# No use getting a lifetime because this credential is
+		# ephemeral, but if you're looking at this to see how to
+		# create a credential, you could also pass the lifetime and
+		# algorithm. Neither of those settings are exposed in the
+		# settings API at the moment.
+		return TokenCredentials(key=keyPath, key_id=keyId, team_id=teamId)
 
 
 def apns_send_message(
-	registration_id: str,
-	alert: Union[str, Alert],
-	application_id: str = None,
-	creds: Credentials = None,
-	topic: str = None,
-	badge: int = None,
-	sound: str = None,
-	extra: dict = {},
-	expiration: int = None,
-	thread_id: str = None,
-	loc_key: str = None,
-	priority: int = None,
-	collapse_id: str = None,
-	err_func: ErrFunc = None,
+		registration_id: str,
+		alert: Union[str, Alert],
+		application_id: str = None,
+		creds: Credentials = None,
+		topic: str = None,
+		badge: int = None,
+		sound: str = None,
+		extra: dict = {},
+		expiration: int = None,
+		thread_id: str = None,
+		loc_key: str = None,
+		priority: int = None,
+		collapse_id: str = None,
+		err_func: ErrFunc = None,
```

Comment:
was this spacing intentional?

Reply:
Not really, updated with 1 tab spacing for all function arguments